cwltool/workflow_job.py

```diff
@@ -60,7 +60,12 @@ def job(
     ) -> JobsGeneratorType:
         runtimeContext = runtimeContext.copy()
         runtimeContext.part_of = self.name
-        runtimeContext.name = shortname(self.id)
+
+        if runtimeContext.workflow_job_step_name_callback is not None:
+            vfinputs = {shortname(k): v for k, v in joborder.items()}
+            runtimeContext.name = runtimeContext.workflow_job_step_name_callback(self, vfinputs)
```

Comment:
No test coverage?

Reply:
I don't understand.  There is a test `test_workflow_job_step_name_callback` and codecov says "All modified and coverable lines are covered by tests"

Reply:
![image](https://github.com/user-attachments/assets/57d7109a-60ee-4f64-af49-c8e794e2f908)

https://github.com/common-workflow-language/cwltool/pull/2109/files#diff-ed87d61e2237460638e7f3ddb21bafc9969d8c142c804d79f94801142342f3d6

Reply:
codecov was out of date, I hit "refresh" and now it says 100% coverage?